BackEnd/utils/image_moderation.py

```diff
@@ -0,0 +1,32 @@
+from django.utils.timezone import now
+from utils.completeness_counter import completeness_count
+from images.models import ProfileImage
+
+
+class ModerationManager:
+    def __init__(self, profile):
+        self.profile = profile
+        self.moderation_is_needed = False
+
+    def update_image(self, image, image_type):
+        existing_image = ProfileImage.objects.filter(
+            hash_md5=image.hash_md5, is_approved=True
+        ).first()
+        if existing_image:
+            image.is_approved = True
+            image.save()
+            setattr(self.profile, f"{image_type}_approved", image)
+            completeness_count(self.profile)
```

Comment:
You will run completeness_count for banner and logo without validation, i think this is not fair, because those images can be declined.

Reply:
We have temporarily decided to omit the logic for re-validating images that have already been approved.